tsconfig.json

```diff
@@ -17,14 +17,17 @@
       "@rainbow-me/config": ["./src/config"],
       "@rainbow-me/entities": ["src/entities"],
       "@rainbow-me/handlers": ["./src/handlers"],
+      "@rainbow-me/handlers/*": ["./src/handlers/*"],
```

Comment:
Just a note: when in a `ts` file and trying to import subdirectories like `@rainbow-me/handlers/uniswap`, it was showing an error: `Cannot find module '@rainbow-me/handlers/uniswap' or its corresponding type declarations.` unless the `/*` path was included in `tsconfig.json`